src/System.Net.Ping/tests/FunctionalTests/System.Net.Ping.Functional.Tests.csproj

```diff
@@ -34,7 +39,7 @@
       <Link>Common\tests\System\PlatformDetection.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'==''">
     <ProjectReference Include="..\..\pkg\System.Net.Ping.pkgproj">
```

Comment:
This pkgproj ProjectReference should work for both 1.3 and 1.7 configurations. Why did you need to move to a package reference for some cases.